FindClang.cmake

```diff
@@ -80,6 +80,14 @@ if (Clang_CODECHECKER_EXECUTABLE STREQUAL "Clang_CODECHECKER_EXECUTABLE_NOTFOUND
   message(WARNING "Could not find codechecker, which is system-dependent. See https://codechecker.readthedocs.io/en/latest/#install-guide")
 endif()
 
+# Need to add symlink to executable because command `clang` does not exist
+if(${CMAKE_HOST_SYSTEM_NAME} STREQUAL "Linux")
+  execute_process(
+    WORKING_DIRECTORY /usr/bin
```

Comment:
ln command can be run from anywhere (think its added to $PATH). maybe stay out of /usr/bin and just give complete paths in ln command

Reply:
should we be checking if clang bin does already exist?

Reply:
Instead of doing this. I am just specifying the correct executable. See my subsequent commit